functions-and-scope.js

```diff
@@ -71,8 +117,15 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 9
-
-
+function highestGrade(array) {
+    if (toString.call(array) !== "[object Array]")
+        return "Dit is geen array, Zet ze even tussen blokhaken!";
+    return Math.max.apply(null, array);
```

Comment:
Hier kun je dit doen
```suggestion
    return Math.max(...array);
```